sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/MiscBenchmark.scala

```diff
@@ -227,4 +227,30 @@ class MiscBenchmark extends BenchmarkBase {
     generate stack wholestage on                   836 /  847         20.1          49.8      15.5X
      */
   }
+
+  ignore("generate explode big struct array") {
+    val N = 60000
+
+    val spark = sparkSession
+    import spark.implicits._
+    import org.apache.spark.sql.functions._
+
+    val df = sparkSession.sparkContext.parallelize(
```

Comment:
please make it a real benchmark
```
val df = Seq("1", Array.fill(N)(i => (i.toString, (i + 1).toString, (i + 2).toString, (i + 3).toString))).toDF("col", "arr")
runBenchmark("generate big struct array", N) {
  df.withColumn("arr_col", explode('arr)).select("col", "arr_col.*").count
}
```

Reply:
and put the result like other benchmarks in this file, e.g.
```
/*
     Java HotSpot(TM) 64-Bit Server VM 1.8.0_92-b14 on Mac OS X 10.11.6
     Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
 
     generate stack:                          Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
     ------------------------------------------------------------------------------------------------
     generate stack wholestage off               12953 / 13070          1.3         772.1       1.0X
      generate stack wholestage on                   836 /  847         20.1          49.8      15.5X
       */
```
You can run this benchmark without your PR and put the result in PR comment